mmv1/templates/terraform/pre_create/alloydb_instance.go.erb

```diff
@@ -0,0 +1,18 @@
+// Read the config and call createsecondary api if instance_type is SECONDARY
+
+var instanceType interface{}
+var cluster interface{}
```

Comment:
Unless I'm missing something, we're never using this value?

Reply:
Removed unused code